bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryParam.scala

```diff
@@ -80,6 +80,16 @@ object InvestigatorInfo {
   }
 }
 
+sealed case class PartnerInfo(name: String, hoursAllocated: Double)
```

Comment:
Since you can extend case classes, it's probably a good idea to mark them `final` though.